roles/cluster_logging/files/kibana/provision-index.sh

```diff
@@ -6,9 +6,11 @@
 
 set -ue
 
-INDEX_ID="logstash-*"
+if [ ! -z "$INDEX_PATTERN" ]; then
```

Comment:
Pretty sure this doesn't work with `set -u` :smiley: 

Also, no real reason for this... `logstash-*` is not some 'magically correct' value. Rather have the whole script fail when a required 'config value' is not set.